src/css/cs/items/_aa_planner.scss

```diff
@@ -16,6 +16,10 @@
           display: none;
         }
       }
+      
+      .ps_box-longedit {
+        width: 300px;
```

Comment:
@darrylumich fine with me, just make sure this works on smaller screens as well. Usually we use `max-width` instead.